### PR TITLE
Added homework_7_261122 (Tasks 1-4)

### PR DESCRIPTION
**Pull request by kate-lvlup**
Hi Dima,
Hope you are well!
Could you check my homework7?

Thanks in advance!

### CODE REVIEW

File: homework_7_261122/src/cars/Car.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    public Car(int maxSpeed, int fuelAmount, int productionYear, double price, String brand) {
+        this.maxSpeed = maxSpeed;
+        this.fuelAmount = fuelAmount;
+        this.productionYear = productionYear;
+        this.price = price;
+        this.brand = brand;
+    }
+
+    public void getMaxSpeed() {
+        System.out.println("Max speed: " + maxSpeed);
```
Comment (YeaaaaH) [inline homework_7_261122/src/cars/Car.java:21]: you need to return a variable not to print it, it relates to all get methods

File: homework_7_261122/src/cars/Car.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    }
+
+    public void getIsEngineStarted() {
+        System.out.println("Is Engine Started: " + isEngineStarted);
+    }
+
+    public void startStopEngine() {
+        if (isEngineStarted) {
+            isEngineStarted = false;
+            currentSpeed = 0;
```
Comment (YeaaaaH) [inline homework_7_261122/src/cars/Car.java:55]: you don't need this line

File: homework_7_261122/src/cars/Car.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        System.out.println("Current speed: " + currentSpeed);
+    }
+
+    public void getIsEngineStarted() {
+        System.out.println("Is Engine Started: " + isEngineStarted);
+    }
+
+    public void startStopEngine() {
+        if (isEngineStarted) {
+            isEngineStarted = false;
```
Comment (YeaaaaH) [inline homework_7_261122/src/cars/Car.java:54]: you don't need this line, engine is started, so it's true

File: homework_7_261122/src/cars/Car.java [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    public void getCurrentSpeed() {
+        System.out.println("Current speed: " + currentSpeed);
+    }
+
+    public void getIsEngineStarted() {
+        System.out.println("Is Engine Started: " + isEngineStarted);
+    }
+
+    public void startStopEngine() {
+        if (isEngineStarted) {
```
Comment (YeaaaaH) [inline homework_7_261122/src/cars/Car.java:53]: this veriable means that engine is started so it's true

File: homework_7_261122/src/cars/Car.java [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+    public void getCurrentSpeed() {
+        System.out.println("Current speed: " + currentSpeed);
+    }
+
+    public void getIsEngineStarted() {
+        System.out.println("Is Engine Started: " + isEngineStarted);
+    }
+
+    public void startStopEngine() {
```
Comment (YeaaaaH) [inline homework_7_261122/src/cars/Car.java:52]: by logic of this method you either start up or shut down engine, better separate it in two methods for start up and shutdown

File: homework_7_261122/src/laptops/Laptop.java [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+    public Laptop(String brand, boolean isTurnedOn, String welcomePhrase, int butteryAmount) {
+        this.brand = brand;
+        this.isTurnedOn = isTurnedOn;
+        this.welcomePhrase = welcomePhrase;
+        this.butteryAmount = butteryAmount;
+    }
+
+    public void getBrand() {
+        System.out.println("Brand: " + brand);
```
Comment (YeaaaaH) [inline homework_7_261122/src/laptops/Laptop.java:17]: return variable, not print it

File: homework_7_261122/src/products/Product.java [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+    protected boolean withBone;
+
+    public Product(int weight, int size, String color, boolean withBone) {
+        this.weight = weight;
+        this.size = size;
+        this.color = color;
+        this.withBone = withBone;
+    }
+
+    public void growInUkraine() {
```
Comment (YeaaaaH) [inline homework_7_261122/src/products/Product.java:18]: name of this method is not clear, what this method should do?

File: homework_7_261122/src/triangles/Triangle.java [hunk 8]
```diff
@@ -0,0 +1,51 @@
+package triangles;
+
+public class Triangle {
```
Comment (YeaaaaH) [inline homework_7_261122/src/triangles/Triangle.java:3]: you need to add constuctor and call verifyPythagoras() method, before creation

File: homework_7_261122/src/cars/Car.java (hunk 2 above)
Comment (kate-lvlup) [reply homework_7_261122/src/cars/Car.java:55]: Fixed. Deleted line 'currentSpeed = 0;'

File: homework_7_261122/src/laptops/Laptop.java (hunk 6 above)
Comment (kate-lvlup) [reply homework_7_261122/src/laptops/Laptop.java:17]: Fixed.

File: homework_7_261122/src/triangles/Triangle.java (hunk 8 above)
Comment (kate-lvlup) [reply homework_7_261122/src/triangles/Triangle.java:3]: Fixed. Created constuctor with all sides and a condition to test the Pythagorean theorem.

File: homework_7_261122/src/cars/Car.java (hunk 1 above)
Comment (kate-lvlup) [reply homework_7_261122/src/cars/Car.java:21]: Fixed.

File: homework_7_261122/src/products/Product.java [hunk 9]
```diff
@@ -0,0 +1,37 @@
+package products;
+
+import java.sql.SQLOutput;
```
Comment (YeaaaaH) [inline homework_7_261122/src/products/Product.java:3]: remove unused import

File: homework_7_261122/src/triangles/Triangle.java [hunk 10]
```diff
@@ -5,28 +5,27 @@ public class Triangle {
...
-    }
-
-    public void getSideA() {
-        System.out.println("Side A: " + sideA);
-    }
-
-    public void setSideB(int sideB) {
         this.sideB = sideB;
+        this.sideC = sideC;
+        if (Math.pow(sideA, 2) + Math.pow(sideB, 2) == Math.pow(sideC, 2)) {
```
Comment (YeaaaaH) [inline homework_7_261122/src/triangles/Triangle.java:12]: use method verifyPythagoras() instead

File: homework_7_261122/src/products/Product.java (hunk 9 above)
Comment (kate-lvlup) [reply homework_7_261122/src/products/Product.java:3]: Fixed.

File: homework_7_261122/src/triangles/Triangle.java (hunk 10 above)
Comment (kate-lvlup) [reply homework_7_261122/src/triangles/Triangle.java:12]: Fixed.

File: homework_7_261122/src/products/Product.java (hunk 9 above)
Comment (kate-lvlup) [reply homework_7_261122/src/products/Product.java:3]: I wonder where it might have come from? (import java.sql.SQLOutput;)

Comment (YeaaaaH) [reply homework_7_261122/src/products/Product.java]: Who knows:)

**Comment by kate-lvlup** [comment]
Branch homework_7_261122 was merged into branch master.

**Pull request merged**